README.md

```diff
@@ -341,7 +368,7 @@ This option is needed to know which attribute should be passed to the parameter
 
 After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release` (this will create a git tag for the version). Push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
```

Comment:
`bundle exec rake release` will not only `create a git tag` but also push `git commits`, `git tags` and `.gem files`.

Reply:
To install this gem onto your local machine, run `bundle exec rake install`. This command will create a git tag for the version, push the git commits, tags and the `.gem` files to [rubygems.org](https://rubygems.org).). To release a new version, update the version number in `version.rb` before running `bundle exec rake release`. 

Does this sound better?

Reply:
It is confusing which command will create a git tag. `rake install` or `rake release`?
How about putting it to the end?

--
To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb` before running `bundle exec rake release` (This command will create a git tag for the version, push the git commits, tags and the `.gem` files to [rubygems.org](https://rubygems.org)).